<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MakePost</class>
 <widget class="QWidget" name="MakePost">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>977</width>
    <height>547</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MakePost</string>
  </property>
  <widget class="QGroupBox" name="postArea">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>60</y>
     <width>601</width>
     <height>331</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox {
background-color:rgb(227, 225, 255);
    border: 3px solid rgb(187, 187, 255);
    border-radius: 9px;

}</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLineEdit" name="makePostText">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>541</width>
      <height>141</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QLabel" name="statusPostMessage">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>260</y>
      <width>471</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
background-color:  rgb(30, 101, 255);
color:rgb(255, 255, 255);
border-radius: 15px;

hover{
background-color: rgb(1, 221, 255);
color:rgb(0, 0, 0);
}
</string>
    </property>
    <property name="text">
     <string>We are waiting for you to write a post!</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="usernameLabel">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>30</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size:19px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="nameInitLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
 border-style: outset;
 border-width: 2px;
 border-color: black;
border-radius:25px;
font: 20pt &quot;Lucida Calligraphy&quot;;
color:black;
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="char_counter">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>60</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="makePostButton">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>450</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{

background-color:  rgb(30, 101, 255);
color:rgb(255, 255, 255);
border-radius: 15px;

}
QPushButton:hover{
background-color: rgb(1, 221, 255);
color:rgb(0, 0, 0);
}
QPushButton:pressed{
background-color:rgb(43, 107, 255)
}</string>
   </property>
   <property name="text">
    <string>Post your Tweet!</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>-5</y>
     <width>991</width>
     <height>561</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(213, 229, 255)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>postArea</zorder>
  <zorder>makePostButton</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
